src/utils/line.ts

```diff
@@ -250,11 +256,49 @@ export const formatAggregatedCreditLineData = (
     return { ...r, [r.token]: (r.totalVolumeUsd ?? '0').toString() };
   }, {});
 
+  const formattedPositions = positions.reduce(
+    (obj: any, c: BasePositionFragResponse): { [id: string]: CreditPosition } => {
+      const {
+        dRate,
+        fRate,
+        id,
+        lender,
+        token,
+        ...financials
+        // events: graphEvents,
+      } = c;
+
+      const currentUsdPrice = tokenPrices[c.token?.id];
+      // const events = graphEvents ? formatCreditEvents(c.token.symbol, currentUsdPrice, graphEvents!) : [];
+      // creditEvents.concat(events);
+      // this formatting does not return same data structure as formatUserPortfolio
+      return {
+        ...obj,
+        [id]: {
+          id,
+          // lender: lender.id,
+          lender,
+          ...financials,
+          dRate,
+          fRate,
+          token: _createTokenView(token, BigNumber.from(principal), currentUsdPrice),
+          // events,
+        },
+      };
+    },
+    {}
+  );
+
+  console.log('formatted page positions', formattedPositions, positions);
+
   return {
     credit: {
       highestApy,
       principal: parseUnits(unnullify(credit.principal), 'ether').toString(),
       deposit: parseUnits(unnullify(credit.deposit), 'ether').toString(),
+      interest: '0', // TODO
```

Comment:
interest and principal aren't being aggregated

Reply:
There is another ticket to rename `AggregatedCreditLine` to `SecuredLine` among other naming conventions for types. Will ignore this for now: 
https://www.notion.so/Consolidate-types-CreditLine-ts-576af5073a9a44089dec8c673f0dba02